packages/core/src/Stores/client-store.js

```diff
@@ -715,6 +716,10 @@ export default class ClientStore extends BaseStore {
         return this.account_status?.status?.includes('financial_assessment_not_complete');
     }
 
+    get is_financial_assessment_needed() {
+        return this.account_status?.status?.includes('financial_assessment_notification');
```

Comment:
Backend checks all the required conditions and sends this single status `financial_assessment_notification` to FE.

Reply:
What is the difference b/w the new status and `financial_information_not_complete` & `financial_assessment_not_complete`?

Reply:
same question.
we also have a notification based on the flag **financial_information_not_complete**.
Do we need both?

Reply:
@likhith-deriv @amina-deriv As per info from BE team, FA is Trading Experience + Financial Information . some clients require both TE and FI for FA to be complete some do not. to avoid confusion, a new status is created for the clients with following conditions:
- low-risk country
- fully authenticated
- MT5 account created in svg
- FA not completed before

Reply:
If I remember correctly, Ragheb said `financial_assessment_not_complete` is both FA+TA and `financial_information_not_complete` is only FA. Can u confirm this with him? And also based on info u get can u please add some info in clinet_stores for all these status. Might help in future

Reply:
BE informed that it would be better to add a seprate flag. the logic for `financial_information_not_complete` and `financial_assessment_not_complete` is already crowded.
And here are the files shared to look into the backend code:
FI: https://github.com/regentmarkets/bom-user/blob/master/lib/BOM/User/Client.pm#L725
FA: https://github.com/regentmarkets/bom-user/blob/master/lib/BOM/User/Client.pm#L684